src/parser/jobs/brd/modules/Snapshots.tsx

```diff
@@ -0,0 +1,260 @@
+import React from 'react'
+import {Trans, Plural} from '@lingui/react'
+import {t} from '@lingui/macro'
+import {Table, Accordion} from 'semantic-ui-react'
+
+import Module, {dependency} from 'parser/core/Module'
+import {CastEvent} from 'fflogs'
+import STATUSES, {Status} from 'data/STATUSES'
+import {SNAPSHOT_BLACKLIST} from 'parser/jobs/brd/modules/SnapshotBlacklist'
+import ACTIONS from 'data/ACTIONS'
+import {ActionLink, StatusLink} from 'components/ui/DbLink'
+import NormalisedMessage from 'components/ui/NormalisedMessage'
+import Combatants from 'parser/core/modules/Combatants'
+import {AbilityEvent} from 'fflogs'
+import Util from './Util'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+
+import styles from './Snapshots.module.css'
+
+const SNAPSHOTTERS = [
+	ACTIONS.IRON_JAWS.id,
+	ACTIONS.CAUSTIC_BITE.id,
+	ACTIONS.STORMBITE.id,
+]
+
+const DOT_STATUSES = [
+	STATUSES.CAUSTIC_BITE.id,
+	STATUSES.STORMBITE.id,
+]
+
+const PERSONAL_STATUSES = [
+	STATUSES.MEDICATED.id,
+	STATUSES.RAGING_STRIKES.id,
+]
+
+const BAD_REFRESH_SEVERITY = {
+	1: SEVERITY.MINOR,
+	2: SEVERITY.MEDIUM,
+	3: SEVERITY.MAJOR,
+}
+
+interface Snapshot {
+	// The event that applied the DoT(s).
+	snapEvent: CastEvent
```

Comment:
This could prob be called `snapshotter` or something, to be more aligned with table output and easier to read.